vnext/Desktop/WebSocketResourceFactory.cpp

```diff
@@ -16,7 +16,7 @@ namespace Microsoft::React {
 /*static*/
 shared_ptr<IWebSocketResource>
 IWebSocketResource::Make(const string &urlString, bool legacyImplementation, bool acceptSelfSigned) {
-  if (!legacyImplementation) {
+  if (false) {
     std::vector<winrt::Windows::Security::Cryptography::Certificates::ChainValidationResult> certExceptions;
```

Comment:
Is this meant to be left in?

Reply:
No. Forgot to leave a comment. It's for CI validation only.

It WILL be reverted.